stock_prices/stock_prices.py

```diff
@@ -2,8 +2,47 @@
 
 import argparse
 
+
 def find_max_profit(prices):
-  pass
+    # min_buy = min(prices)
+    # max_sell = max(prices)
+    # min_buy = 10000
+    # max_buy = [0, 2]
+    # index = 0
+    # for i in prices:
+    #
+    #     if i > max_buy[0] and index > 0:
+    #         max_buy[0] = i
+    #         max_buy[1] = index
+    #         print("MAX", max_buy[0])
+    #
+    #     if i < min_buy and index < max_buy[1]:
+    #         min_buy = i
+    #         print("MIN:", min_buy)
+    #     index += 1
+    #
+    # return max_buy[0] - min_buy
+    max_profit = -10 # Yes there was a work around
+    min_buy = 0
+    sell_price = 0
+    change_buy_index = True
+
+    for i in range(0, len(prices) - 1):  # this is so i doesn't go out of range on line 31
+      sell_price = prices[i + 1]
+      if change_buy_index:
+        min_buy = prices[i]
+      if sell_price < min_buy:
+        change_buy_index = True
+        continue
+      else:
```

Comment:
this is a strange if trail. You can just use if elif to remove the continue here. 
But it's not necessary since you passed all of the tests, just getting coding practices.
Good Job here Drew!